locallib.php

```diff
@@ -2626,6 +2626,38 @@ function theme_boost_union_is_active_theme() {
     }
 }
 
+/**
+ * Helper function to generate HTML for an alert when Boost Union is not the active theme,
+ * but someone tries to access Boost Union's settings.
+ *
+ * @return string HTML for the alert.
+ */
+function theme_boost_union_is_not_active_alert() {
+    global $SCRIPT, $OUTPUT;
```

Comment:
`$SCRIPT` is a Moodle global variable, but I do not think that you will need it here.I will remove that in another review changes commmit